docs/api-reference/regularization-l1-l2.md

```diff
@@ -0,0 +1,27 @@
+This class uses [empricial risk minimization](https://en.wikipedia.org/wiki/Empirical_risk_minimization) (i.e., ERM)
```

Comment:
>This  [](start = 0, length = 5)

Please add a header '### Regularization' so that the following text becomes a separate section. Also move it after all the algo details.

Reply:
No. I don't only mean regularization. It is a brief introduction to the whole optimization problem.

---
In reply to: [284914354](https://github.com/dotnet/machinelearning/pull/3586#discussion_r284914354) [](ancestors = 284914354)